scripts/mirror-provider.sh

```diff
@@ -1,16 +1,21 @@
 #!/usr/bin/env bash
 
+# create a .terraformrc file only if the file is not present
 write_terraform_rc() {
-  cat > ~/.terraformrc << EOL
+  if [[ -f "$HOME/.terraformrc" ]]; then
+    return 0
+  fi
+
+  cat > "$HOME/.terraformrc" << EOL
 provider_installation {
-            filesystem_mirror {
-              path    = "/root/.terraform.d/plugins"
-              include = ["lacework/lacework"]
-            }
-            direct {
-              exclude = ["lacework/lacework"]
-            }
-          }
+    filesystem_mirror {
+        path    = "$HOME/.terraform.d/plugins"
```

Comment:
**It took me a while to figure this one out** 

![tenor-133131062](https://user-images.githubusercontent.com/5712253/127073800-9bc51e06-6371-48e1-b3bd-35998406e73c.gif)